secret/src/main/java/com/sequenceiq/secret/SecretAspects.java

```diff
@@ -0,0 +1,57 @@
+package com.sequenceiq.secret;
+
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Pointcut;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sequenceiq.secret.service.SecretAspectService;
+import com.sequenceiq.secret.service.SecretService;
+
+//@Component
+//@Aspect
```

Comment:
Will aspects work without this annotation?

Reply:
Yes, it will. this is just an example. Someone who wants to use the secret module, they should create the proper `@Aspect`